db.go

```diff
@@ -291,13 +292,7 @@ func (s *ColumnStore) recoverDBsFromStorage(ctx context.Context) error {
 		databaseName := f.Name()
 		g.Go(func() error {
 			// Open the DB for the side effect of the snapshot and WALs being loaded as part of the open operation.
-			_, err := s.DB(
-				ctx,
-				databaseName,
-				WithCompactionAfterOpen(
```

Comment:
Why are we removing compaction after recovery?

Reply:
Actually not sure it will be necessary or that effective since compaction should now be triggered on writes in replays, right?